03-nodejs/00-learn-nodejs-basics/01-content-negotiation/requestForAuthorInfo.js

```diff
@@ -1 +1,24 @@
-// Your code here
\ No newline at end of file
+// Your code here
+var http = require("http");
+var reqTypes = ["text/plain", "text/html", "application/json"];
+
+for (var i = 0; i < reqTypes.length; i++) {
+    var request = http.request({
+        hostname: "eloquentjavascript.net",
+        path: "/author",
+        method: "GET",
+        headers: {
+            Accept: reqTypes[i]
+        }
+    }, function (response) {
+        console.log("Status code: ", response.statusCode);
+        response.on("data", function (chunk) {
+            console.log(chunk.toString());
+        });
+        response.on("end", function () {
+            return;
+        });
+    });
+
+    request.end();
+}
```

Comment:
Provide new line in the end of each file

For more info 

http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline